### PR TITLE
Feature comparison page

### LINKED ISSUE #30: Create comparison page
This page shows us the difference between chosen pets and helps to make the best choice

### PR DESCRIPTION
**Pull request by ivan-bonk**
- add shared logotype component
- add saga and redux actions for fetching data from DB
- add add additioal component Rate for showing right count and color
- add router links for redirecting to pet page

### CODE REVIEW

**Review by Pashkaua — APPROVED** [review]
as for me
looks good

**Review by Danik1702 — APPROVED** [review]
Pretty good!

**Review by Beker94 — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -5,39 +5,10 @@
...
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <title>My Pet</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (okhabr) [inline public/index.html:30]: I'm not quite sure if it is a good idea to remove <noscript> tag

File: src/pages/comparison/components/rate.component.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    return (
+        <span className="rate-bar">
+            <div className={clases[0]}></div>
+            <div className={clases[1]}></div>
+            <div className={clases[2]}></div>
+            <div className={clases[3]}></div>
+            <div className={clases[4]}></div>
+        </span>
```
Comment (okhabr) [inline src/pages/comparison/components/rate.component.tsx]: Seems that divs inside span is not the best choice.
Would you like to consider using loop in this case?

File: src/pages/comparison/components/characteristics.component.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    const { pets, display, rate } = props;
+
+    const items = pets.map((el: Pet) => {
+        const property = Object.getOwnPropertyDescriptor(el, display);
+
+        const show = rate ? <Rate rate={property?.value} /> : <span>{property?.value}</span>;
+
+        const styles = {
+            color: 'black',
+            textDecoration: 'none'
```
Comment (okhabr) [inline src/pages/comparison/components/characteristics.component.tsx]: Not sure if it is a best practice to treat css styling like that, maybe it is better to use styles.css for styling

**Review by okhabr — APPROVED** [review]
It took some google researching for me to go through some parts of your code :) (It's recognition, btw)
I've left few comments for your consideration:)

File: public/index.html (hunk 1 above)
Comment (ivan-bonk) [reply public/index.html:30]: Yes, I agree with you
Always forget about browsers without script supporting. :(

File: src/pages/comparison/components/characteristics.component.tsx (hunk 3 above)
Comment (ivan-bonk) [reply src/pages/comparison/components/characteristics.component.tsx]: Thanks, i will try to solve this issue

File: src/pages/comparison/components/rate.component.tsx (hunk 2 above)
Comment (ivan-bonk) [reply src/pages/comparison/components/rate.component.tsx]: After so much work on the React, you no longer pay attention to html :)

Actually i had no idea how to use loop in this case, but i will think about it. Maybe you solve this issue much better :)

**Pull request merged**